### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/notalone#45]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://www.notalone.gov:
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - #escape
  - `<a class="btn btn-escape" href="https://www.google.com" id="escape">Escape</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(3) > div > div > div > a
  - `<a class="btn action" href="#find-box">Explore</a>`
- **Error:** Form does not contain a submit button (input type="submit", input type="image", or button type="submit").
  - WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
  - #search-form
  - `<form role="form" id="search-form" class="js-required" action="/search/" method="GET"> <label class="sr-onl...</form>`
- **Error:** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - html > body > div:nth-child(5) > div > div > div > div:nth-child(7) > iframe
  - `<iframe src="//www.youtube-nocookie.com/embed/xLdElcv5qqc" frameborder="0" allowfullscreen=""></iframe>`
- **Error:** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - html > body > div:nth-child(5) > div > div > div > div:nth-child(9) > iframe
  - `<iframe src="//www.youtube-nocookie.com/embed/wNMZo31LziM" frameborder="0" allowfullscreen=""></iframe>`
- **Error:** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - html > body > div:nth-child(5) > div > div > div > div:nth-child(11) > iframe
  - `<iframe src="//www.youtube-nocookie.com/embed/XXox6ma1gtE" frameborder="0" allowfullscreen=""></iframe>`
- **Error:** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - html > body > div:nth-child(5) > div > div > div > div:nth-child(13) > iframe
  - `<iframe src="//www.youtube-nocookie.com/embed/sTHNAwCU-bk" frameborder="0" allowfullscreen=""></iframe>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(3) > a
  - `<a class="btn btn-xs" href="https://ohl.rainn.org/online">Live Chat</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(1) > div:nth-child(3) > div:nth-child(4) > a
  - `<a class="btn btn-xs" href="http://www.loveisrespect.org/get-help/contact-us/chat-with-us/">Live Chat</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.1:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(1) > div:nth-child(4) > div:nth-child(3) > a
  - `<a class="btn btn-xs" href="http://www.thehotline.org/what-is-live-chat/">Live Chat</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(2) > div > div:nth-child(1) > div > a
  - `<a href="/students/">Students</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(2) > div > div:nth-child(2) > div > a
  - `<a href="/schools/">Schools</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(2) > div > div:nth-child(3) > div > a
  - `<a href="/resources/">Resources</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(1) > div:nth-child(2) > div > div:nth-child(4) > div > a
  - `<a href="/data/">Data</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.74:1. Recommendation: change background to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div:nth-child(5) > footer > div:nth-child(2) > div:nth-child(1) > a
  - `<a class="btn btn-lg cta" href="/contact">Contact</a>`
- **Error:** Form does not contain a submit button (input type="submit", input type="image", or button type="submit").
  - WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
  - #footer-search-form
  - `<form role="form" id="footer-search-form" class="js-required" action="/search/" method="GET"> <label class="sr-o...</form>`
- **Error:** Duplicate id attribute value "q" found on the web page.
  - WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
  - #q
  - `<input type="search" class="form-control" id="q" name="q" placeholder="Search">`
## Summary:
- 17 Errors
